### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by 89z** [alecthomas/chroma#463]
Can you please push new release? I am using Dev version of Hugo, because my commit here was just after the last release in November

thank you!

**Comment by cgay** [comment]
Plus one, I would love to have my Dylan lexer contribution be upstreamed into Hugo (for which I assume a new Chroma release is needed). Thanks!

**Comment by walles** [comment]
@cgay it might be that the Hugo people could just do this:

```
go get github.com/alecthomas/chroma@eb257ed1bb1c36eb819a4ecf720bb343348b6ace
```

That should get them a newer Chroma without any release.

**Comment by cgay** [comment]
(It's not urgent for me at all, just a nice-to-have.)

**Comment by alecthomas** [comment]
Released v0.9.0.
